### PR TITLE
feat: header & search component staged

### PR DESCRIPTION
**Pull request by codesnail21**
basic header component is staged with search bar

### CODE REVIEW

File: .gitconfig [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[user]
+    email = vatsal@improwised.com
+    name = codesnail21
```
Comment (zeelrupapara) [inline .gitconfig:3]: add line please

File: components/TheHeader.vue [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="head navbar align-content-center">
+    <h1>Where in the world?</h1>
```
Comment (zeelrupapara) [inline components/TheHeader.vue:3]: where is a moon icon?

File: components/TheSearch.vue [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        class="search p-1 w-25 mx-4 mt-5 p-3 w-25"
+        placeholder="Search for a country..."
+        type="text"
+      />
+      <select
+        id=""
+        class="options mr-4 px-5 my-4 border-0 bg-white shadow w-30 font-font-weight-normal"
+      >
+        <option value="volvo">Filter By Region</option>
+        <option value="saab">Saab</option>
```
Comment (zeelrupapara) [inline components/TheSearch.vue:14]: what is a saab, audi and ....other

**Review by zeelrupapara — CHANGES_REQUESTED** [review]

File: components/TheHeader.vue (hunk 2 above)
Comment (codesnail21) [reply components/TheHeader.vue:3]: adding it last

File: components/TheSearch.vue (hunk 3 above)
Comment (codesnail21) [reply components/TheSearch.vue:14]: thats test items

File: .gitconfig (hunk 1 above)
Comment (codesnail21) [reply .gitconfig:3]: okay

**Review by Payal17122000 — APPROVED** [review]

**Review by zeelrupapara — APPROVED** [review]

**Pull request merged**